src/practitest_firecracker/practitest.clj

```diff
@@ -161,8 +171,15 @@
                                       :attributes attributes
                                       :instances  {:test-ids test-ids}}}}))))
 
+(defn make-instances [testset-tests]
+  (for [[testset-id test-ids] testset-tests]
+    (for [test-id test-ids]
+      {:type       "instances"
+       :attributes {:set-id testset-id
+                    :test-id test-id}})))
```

Comment:
There is no need to use nested `for`, just do it like this:

```clojure
(for [[testset-id test-ids] testset-tests
      test-id               test-ids]
  {:type "instances"
   :attributes {:set-id testset-id :test-id test-id})
```

Reply:
wasn't sure if this would work and preferred to go for the safe side.
At the bottom line it will do the same